CMakeLists.txt

```diff
@@ -359,6 +359,8 @@ endif()
 # Library config
 
 set(LIBPARQUET_SRCS
+  src/parquet/types.cc
```

Comment:
I noticed that there are other functions where `inline` might not be helpful. I moved these to a separate file.